docs/0001-publishing.md

```diff
@@ -0,0 +1,47 @@
+---
+Start Date: 2020-07-27
+RFC PR: (leave this empty)
+Foam Issue: (leave this empty)
+---
+
+# 0001 Publishing
```

Comment:
I imagine there are 2 things we are covering in this RFC that are related but maybe not necessarily coupled:
- generating a static website out of a foam (gatsby, jekyll, ...)
- publishing a generated site to a hosting provider (netlify, github pages, ...)

Would it be helpful to split these two aspects?

Reply:
Indeed, there are 2 things, and I wanted to cover them together at a high level to see how they can fit together. As mentioned in the RFC, follow-up RFCs should be opened for specific implementations

Reply:
Understood, I guess I was confused because I saw that it also includes one specific implementation (replacing jekyll w/ gatsby), and looking again at Sanket's comment (whom for some reason I seem unable to @mention) I believe it is along the same lines. 
Anyhow, I like going for gatsby over jekyll!